scripts/update/00-run.sh

```diff
@@ -16,8 +16,12 @@ echo
 
 # Make sure any previous backup doesn't exist
 if [[ -d "$UMBREL_ROOT"/.umbrel-backup ]]; then
-    echo "Cannot install update. A previous backup already exists at $UMBREL_ROOT/.umbrel-backup"
-    echo "This can only happen if the previous update installation wasn't successful"
+    backup_error="Cannot install update. A previous backup already exists at $UMBREL_ROOT/.umbrel-backup"
+    backup_error="$backup_error . This can only happen if the previous update installation wasn't successful."
+    echo "${backup_error}"
+    cat <<EOF > "$UMBREL_ROOT"/statuses/update-status.json
```

Comment:
same here. better enclose the whole path in quotes. it's really just a cosmetic change and not something that would cause any problems

Reply:
This one can be done without breaking the functionality.  Stand by for fix...